### DISCUSSION TITLE
something bad happened

### DISCUSSION THREAD

**Discussion by Utkarsha-Shripanavar** [apache/airflow#36357] [General]
### Apache Airflow version

2.8.0

### What happened?

After running airflow scheduler in jarvis terminal, tried to open endpoint two and got the error like something bad happened.

### How to reproduce

run airflow schedular command 

### Operating System

jarvis

### Deployment

Other

### Are you willing to submit PR?

- [ ] Yes I am willing to submit a PR!

### Code of Conduct

- [X] I agree to follow this project's [Code of Conduct](https://github.com/apache/airflow/blob/main/CODE_OF_CONDUCT.md)

**Comment by boring-cyborg** [comment]
Thanks for opening your first issue here! Be sure to follow the issue template! If you are willing to raise PR to address this issue please do so, no need to wait for approval.

**Comment by potiuk** [comment]
Yeah. Something bad happened. As advised in the error mesage. You need to look at the logs to find out what "bad" happened.

Converting to discussion if needed, but without providing no information, there is absolutely no way anyone will be able to help you wilth your problem.
